### PR TITLE
Completed Step 0

### PR DESCRIPTION
**Pull request by jhileman07**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -26,3 +26,5 @@ int main() {
     QUILL_LOG_INFO(logger, "Current risk held is {}.", riskTracker.getRisk());
     return 0;
 }
+
+// adding this comment to do another pr for ethan
```
Comment (echavemann) [inline src/main.cpp:30]: why

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    "initializer_list": "cpp",
+    "iomanip": "cpp",
+    "iosfwd": "cpp",
+    "iostream": "cpp",
+    "istream": "cpp",
+    "limits": "cpp",
+    "mutex": "cpp",
+    "new": "cpp",
+    "numbers": "cpp",
+    "ostream": "cpp",
```
Comment (echavemann) [inline .vscode/settings.json:63]: gitignore pls

**Review by echavemann — CHANGES_REQUESTED** [review]

**Review by minnce — CHANGES_REQUESTED** [review]
refer to ethan's comments

File: src/main.cpp (hunk 1 above)
Comment (jhileman07) [reply src/main.cpp:30]: i was testing to make sure changes were showing up

File: .vscode/settings.json (hunk 2 above)
Comment (jhileman07) [reply .vscode/settings.json:63]: done

**Review by minnce — APPROVED** [review]
lgtm

**Pull request merged**